modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,229 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+:_content-type: PROCEDURE
+[id="virt-measuring-latency-vm-secondary-network_{context}"]
+= Verifying connectivity and measuring latency between virtual machines on a secondary network
+
+As a cluster administrator, you use a predefined checkup to verify network connectivity and measure latency between virtual machines (VMs) that are attached to a secondary network interface. Set up the namespace and service account for the framework; create the `ClusterRole` and `ClusterRoleBinding` objects for the service account; enable permissions for the checkup; create the input config map; and run the checkup. After you record the results of the checkup, delete all the framework resources that you previously created.
```

Comment:
> Set up the namespace and service account for the framework; create the `ClusterRole` and `ClusterRoleBinding` objects for the service account;

Do we have to give all these details at this stage? IMO it gives a feeling of over-complexity for a standby reader.
Can we say instead "Deploy the framework"?

> enable permissions for the checkup; create the input config map; and run the checkup.

and here: "deploy the checkup (i.e. the permissions it requires); ..."